### ISSUE TITLE
Not working on dokku

### ISSUE THREAD

**Issue by kikonen** [kikonen/ngannotate-rails#26]
From @ulitiy

Not working on dokku either. Even after cleaning the cache and bumping the assets version.

----
Moved here from https://github.com/kikonen/ngannotate-rails/issues/10

**Comment by kikonen** [comment]
Since dokku is different platfoem than heroku, failures there can be non heroku issue related.

Secondly, any traces, logs??

My first suspect with dokku is that needed dependencies for ngannotate are not present. Look into any tracefiles you have from deploy.

**Comment by ulitiy** [comment]
Well it uses heroku buildpacks so problems could be related.
`unknown provider: e` error

**Comment by kikonen** [comment]
Could you try locally with compiled assets to see how it works (see readme)

**Comment by kikonen** [comment]
Updated to latest ngannotate; however, I don't believe it encounters your trouble.

**Comment by kikonen** [comment]
So, what you get if you do this

````
NG_FORCE=true RAILS_ENV=development bundle exec rake assets:clobber assets:precompile
NG_FORCE=true rails s -p 3001
````

**Comment by ulitiy** [comment]
I don't see it working at all)
```
  angular.module('btdDashboard', ['btdDashboardComponents', 'ui.router']).config(function($stateProvider, $urlRouterProvider) {
    $urlRouterProvider.otherwise("/");
    return $stateProvider.state('index', {
      url: '/',
      templateUrl: 'index.html'
    }).state('click', {
      abstract: true,
      url: '/click',
      template: '<ui-view/>'
    }).state('click.before', {
      url: '',
      templateUrl: 'click/before.html'
    }).state('click.count_back', {
      templateUrl: 'click/back_counter.html',
      controller: function($scope, $state, $timeout) {
```

**Comment by kikonen** [comment]
Hm.. should have remembered to notify that for full assets precompile test locally, tou need to modify environments/development.rb. see readme for info.

**Comment by kikonen** [comment]
Q: what traces you see in development.log. there should be some "ng-xxxx" traces there.

**Comment by ulitiy** [comment]
Yes, I installed the github master version and did modifications to the config file, but I don't see any traces though. I can't see anything working at all. Maybe it doesn't work with coffeescript assets? Sorry...

**Comment by kikonen** [comment]
If there is no ng-xxx traces then surely processor is not used at all. It could be related to coffeescript. Have to  admit that I havent tried with it.
